src/test/java/uk/gov/justice/digital/hmpps/cmd/api/repository/NotificationRepositoryTest.kt

```diff
@@ -256,7 +254,6 @@ class NotificationRepositoryTest(
       val actionType = DetailModificationType.ADD
 
       return Notification(
-        id = 1L,
```

Comment:
Maybe Hibernate used to ignore the id because it knows to generate it? But has changed recently to only genereate if not passed? No that doesn't sound right either. 🤔